static/src/js/middleware/metrics/events.js

```diff
@@ -7,6 +7,8 @@ import {
   computeFacets
 } from './helpers'
 
+import staticConfig from '../../../../../static.config.json'
```

Comment:
use sharedUtils/config.js

Reply:
This is a workaround because `config.js` uses lodash's merge to combine the static config and the override, and for some reason, just can't get lodash to play nicely within Playwright tests.

Reply:
Nevermind - I thought I needed this workaround, but that was for a different earlier design where Playwright would have needed to get the URLs for our lambdas. In the current design, only the app code needs to load the `config.js`, which works fine. The latest commit switches this to use `config.js`